### ISSUE TITLE
`cli.isPathIgnored` doesn't work with absolute path

### ISSUE THREAD

**Issue by deepsweet** [eslint/eslint#5138]
```
$ eslint -v
v1.10.3

$ cat .eslintignore
coverage/
```

```js
var CLIEngine = require('eslint').CLIEngine;

var cli = new CLIEngine();

console.log(
    cli.isPathIgnored('coverage/prettify.js'),
    cli.isPathIgnored('/absolute/path/coverage/prettify.js')
);
// true false
```

I'm not 100% sure but in my opinion it's a bug.

**Comment by eslintbot** [comment]
@deepsweet Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
Can you try with the master branch? It might already be fixed.

**Comment by deepsweet** [comment]
there is no such bug with eslint@v2.0.0-rc.0, so I'll wait for major release. feel free to close this if you have no plans to port this into eslint@1.

**Comment by ilyavolodin** [comment]
We are very close to 2.0 release. I think at this point we have no plans of back-porting anything into 1.0 branch. Closing as it looks like it's fixed now.
